sonic_platform_base/sonic_xcvr/api/public/c_cmis.py

```diff
@@ -6,17 +6,19 @@
 from sonic_py_common import logger
 from ...fields import consts
 from .cmis import CmisApi
+import logging
```

Comment:
recommend not having logging here, as it logs on the console, in this case its not going to be caught anywhere inside pmon. 
For this change, you can ignore the exception as pass, Let's try going with the route of passing a logger instance to the constructor, or else
try fixing the sonic_py_common lib function to take the first invocation itself as the final identifier. 
Both ways should work

Reply:
Per chatting with @vdahiya12 offline, since many other places are using logging, let us keep it in use till we decide the better way to handle such log identifier overriding.